core/embed/extmod/modtrezorcrypto/modtrezorcrypto-rfc6979.h

```diff
@@ -40,7 +40,7 @@ STATIC mp_obj_t mod_trezorcrypto_Rfc6979_make_new(const mp_obj_type_t *type,
                                                   size_t n_args, size_t n_kw,
                                                   const mp_obj_t *args) {
   mp_arg_check_num(n_args, n_kw, 2, 2, false);
-  mp_obj_Rfc6979_t *o = m_new_obj(mp_obj_Rfc6979_t);
+  mp_obj_Rfc6979_t *o = m_new_obj_with_finaliser(mp_obj_Rfc6979_t);
```

Comment:
`rfc6979` doesn't have a `__del__()` method. I guess it should have one to memzero the context. However, I can't actually see any use of rfc6979 in our code.

Reply:
Let's drop the thing if it's not being used.

Reply:
done in 6cb8eed